.github/pull_request_template.md

```diff
@@ -6,12 +6,15 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
-
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+Use # to link to relevant PRs if the provided content is not part the present PR.
```

Comment:
To ease confusion we might want to give instruction on what to do with the dangling # symbol if a PR reference is not needed. Remove it or leave it? I don't mind either way, I just think we should make it clear:

```suggestion
Use # to link to relevant PRs if the provided content is not part the present PR. If the provided content is part of the present PR remove the # symbol.
```